test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -1815,4 +1815,26 @@ public void testDeleteObjectWithSubobject() throws Exception {
 
         assertThat("objects not deleted", res2, is(set()));
     }
+
+    @Test
+    public void testSearchObjectType() throws Exception {
+        indexStorage.indexObjects(
+                ObjectTypeParsingRules.getBuilder(
+                        new SearchObjectType("Someobject", 1),
+                        new StorageObjectType("foo", "bar"))
+                        .withIndexingRule(IndexingRules.fromPath(new ObjectJsonPath("whee"))
+                                .build())
+                        .build(),
+                SourceData.getBuilder(new UObject(new HashMap<>()), "objname", "creator").build(),
+                Instant.ofEpochMilli(10000),
+                null,
+                new GUID("WS:2000/1/1"),
+                ImmutableMap.of(new GUID("WS:2000/1/1"), new ParsedObject(
+                        "{\"whee\": \"imaprettypony\"}",
+                        ImmutableMap.of("whee", Arrays.asList("imaprettypony")))),
+                false);
+        final boolean res = indexStorage.lookupParentId("someobject", new GUID("WS:2000/1/1"));
+        assertThat("could not find parent id", res, is(true));
```

Comment:
What I think this test is doing:

Step1) Index an object 
Step2) Check to see if parentID of that guid exists. If its true, that means it has been indexed already.

Should we also have a test that checks to see if an event has not been indexed already?

Reply:
There is already a test in IndexerWorkerTest for that

Reply:
Ok great